test/extensions/filters/http/ext_proc/ext_proc_grpc_fuzz_persistent.cc

```diff
@@ -0,0 +1,133 @@
+#include "test/extensions/filters/http/ext_proc/ext_proc_grpc_fuzz.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace ExternalProcessing {
+
+using envoy::extensions::filters::http::ext_proc::v3::ProcessingMode;
+using envoy::service::ext_proc::v3::ProcessingRequest;
+using envoy::service::ext_proc::v3::ProcessingResponse;
+
+// Using persistent Envoy and ext_proc test server.
+static std::unique_ptr<ExtProcIntegrationFuzz> fuzzer = nullptr;
+static std::unique_ptr<ExtProcFuzzHelper> fuzz_helper = nullptr;
+// Protects fuzz_helper which will be accessed by multiple threads.
+static Thread::MutexBasicLockable fuzz_helper_lock_;
+
+static uint32_t fuzz_exec_count = 0;
```

Comment:
nit: I think the static global vars can probably move to static local vars.

Reply:
Done